emlparser/emlparser.py

```diff
@@ -29,9 +30,18 @@ def json_serial(obj):
             return serial
 
     def execute(self, request):
-        parsed_eml = eml_parser.eml_parser.decode_email_b(request.file_contents,
-                                                          include_raw_body=True,
-                                                          include_attachment_data=True)
+        parser = eml_parser.eml_parser.EmlParser(include_raw_body=True, include_attachment_data=True)
+
+        # Validate URLs in sample, strip out [] if found
+        content_str = request.file_contents.decode(errors="ignore")
+        for u in eml_parser.regex.url_regex_simple.findall(content_str):
+            try:
```

Comment:
What about the case where there is a ValueError and there are no [] in `u`? Also you're running `urlparse(u)` on all hits of the regex looking for an error, so this may be inefficient. Rather than try-catch, maybe use:
```
square_brackets = "[]"
for u in eml_parser.regex.url_regex_simple.findall(content_str):
    if square_brackets in u:
        replacement = u.strip(square_brackets)
        content_str = content_str.replace(u, replacement, 1)
parsed_eml = parser.decode_email_bytes(content_str.encode())
```